fluent-validator/src/main/java/com/baidu/unbiz/fluentvalidator/ValidatorContext.java

```diff
@@ -36,6 +38,19 @@
      */
     public ValidationResult result;
 
+    public boolean validateWith(Function<FluentValidator, FluentValidator> configuration) {
+        ComplexResult localResult = configuration.apply(FluentValidator.checkAll())
```

Comment:
I see your point. Wouldn't it be better if there is a QuickValidator util to leverage? Otherwise the context is polluted with some customized shortcut code template. I think because Java 7 lacks of the ability to pass an "action" to a method, so we have wrap the biz logic in a Function interface to pass in order to not writing the same code as FluentValidator.checkAll().on(...).doValidate.toResult(..),  I think that should be your motivation and reason, is my understanding correct?